lib/new_github_service.rb

```diff
@@ -0,0 +1,77 @@
+module NewGithubService
+  ##
+  # GithubService is miq-bot's interface to the Github API. It acts as a
+  # wrapper around Octokit, delegating calls directly to the Octokit client as
+  # well as providing a space to keep useful augmentations of the interface for
+  # our own use cases.
+  #
+  # You can find the official Octokit documentation at http://octokit.github.io/octokit.rb
+  #
+  # Please check the documentation first before adding helper methods, as they
+  # may already be well handled by Octokit itself.
+  #
+  class << self
+    def service
+      @service ||= \
```

Comment:
Oh weird; I thought in an assignment, if you wanted the value on the line below it, you'd need this.

ie I thought to have a begin block you'd need to do this, or this:  

```ruby
@service ||= begin
  code_that_does_stuff
  more_code
  # ...
```

Neat.